Jenkinsfile

```diff
@@ -190,6 +190,8 @@ stage("Unit Test") {
           # launch different process for each test to make sure memory is released
           python3 -m pip install --upgrade pytest-xdist
 
+          python3 -m pip install 'black~=22.0,>=22.3'
+          black --check --preview --diff text/src/autogluon/text/automm
```

Comment:
Will this fail CI if it doesn't align with black formatting? Should we add a README on this? Unclear currently steps someone needs to take to satisfy formatting in their PR.

Reply:
Let’s add a README in the next PR and merge this in first.